src/en/developer-guide/applications/utilities/create-and-open.md

```diff
@@ -0,0 +1,28 @@
+---
+author: Lukáš Gallovič
+---
+
+# Create And Open - Entity Generator
+
+### This is a tutorial on how to use custom TS code to generate an entity from a different entity.
```

Comment:
This is a tutorial on how to use custom TS code to generate an entity **record from an entity form through ribbon button.**

Reply:
Fixed